weaver/wps_restapi/swagger_definitions.py

```diff
@@ -354,6 +362,13 @@ class URL(ExtendedSchemaNode):
     format = "url"
 
 
+class NRST(ExtendedSchemaNode):
+    schema_type = String
```

Comment:
Add the title `NamespacedRelationshipType` so it gets used in the generated OpenAPI docs.